site/content/en/docs/manual/advanced/formats/format-yolo-ultralytics.md

```diff
@@ -127,6 +127,16 @@ is named to correspond with its associated image file.
 For example, `frame_000001.txt` serves as the annotation for the
 `frame_000001.jpg` image.
 
+### Track support
```

Comment:
Please update information about tracks in the line 33 and in the last column of the format table https://github.com/cvat-ai/cvat/blob/749eb3f771b045ad74ee52beab803f49e953db57/site/content/en/docs/manual/advanced/formats/_index.md#data-export-formats